packages/analytics/lib/index.d.ts

```diff
@@ -667,10 +678,10 @@ export namespace FirebaseAnalyticsTypes {
      *
      * @param screenName A screen name, e.g. Product.
      * @param screenClassOverride On Android, React Native runs in a single activity called
+     * @deprecated
      * 'MainActivity'. Setting this parameter overrides the default name shown on logs.
```

Comment:
This one is still out of order - the lines just need to be flipped so `@deprecated` doesn't break up the multi-line param from above

```suggestion
     *        'MainActivity'. Setting this parameter overrides the default name shown on logs.
     * @deprecated
```